### DISCUSSION TITLE
Error module

### DISCUSSION THREAD

**Discussion by nloshka404** [anope/anope#405] [Support] [answered]
[May 22 10:36:52 2024] Loading modules...
[May 22 10:36:52 2024] Error while loading enc_sha256: enc_sha256 is deprecated and can not be used as a primary encryption method
[May 22 10:36:52 2024] You must load an encryption module!
Why is the module not loading?

**Comment by sadiepowell** [comment, marked as answer]
It says why in the message. You've loaded enc_sha256 as the first password encryption method. This is deprecated so it can't be used as the primary encryption module.

If you're upgrading from 2.0 you need to load another encryption module like enc_sha2 before enc_sha256 so user passwords can be migrated to the new system. Once `/OS STATS PASSWORD` says that no users are using sha256 you can unload enc_sha256`. See [the upgrading guide](https://www.anope.org/upgrading.html) for more information on how to upgrade from 2.0.

If you're not upgrading from 2.0 then you can just replace enc_sha256 with enc_sha2.

**Comment by nloshka404** [comment]
Thanks it worked
